core/themes/basis/css/component/cards.css

```diff
@@ -12,7 +12,17 @@
   padding-bottom: 2rem;
   background-color: #f2f2f2;
 }
-
+body.update-1-30 .l-top .block-views-promoted-cards-block {
+  margin-top: 0;
+}
+body.update-1-30 .l-wrapper > .l-top:has(.block-views-promoted-cards-block) {
+  background-color: #f2f2f2;
+  margin-top: -2rem; /* Negative top margin on parent limited in scope. */
+}
+body.update-1-30 .l-wrapper > .l-top:has(.block-views-promoted-cards-block) .block,
+body.update-1-30 .l-wrapper > .l-top:has(.block-views-promoted-cards-block) .l-messages {
```

Comment:
If I switch that to:
```
body.update-1-30 .l-wrapper > .l-top .block-system-messages:has(+ .block-views-promoted-cards-block) .l-messages
```
it further drills down the bottom margin (0px) to only when the system messages are immediately followed by the cards block.